trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './nameable'
+class TrimmerDecorator < Nameable
+  def correct_name
+    super()
+    @nameable.correct_name.slice(0, 10)
+  end
+end

```

Comment:
- [ ] According to the project requirements, the TrimmerDecorator  should Inherit from `BaseDecorator` instead of the Nameable class
* Check `TrimmerDecorator` class, it should look similar to:

```ruby
# Inherits from Decorator
class CapitalizeDecorator < BaseDecorator
    require './base_decorator'

    # The name has a maximum of 10 characters. If it's longer it should trim the word.`
    def correct_name
      @nameable.correct_name...
end
```